tests.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM vapor/swift
+
+ADD Package.swift Package.swift
+ADD Sources Sources
+ADD Tests Tests
+
+CMD swift test --enable-test-discovery
```

Comment:
I'm not sure what the value in having this dockerfile & the `.tests` compose file are bringing - I'm inclined to just remove them

Reply:
I used them during development. Yes, will delete.